package.json

```diff
@@ -23,8 +23,9 @@
     ]
   },
   "dependencies": {
-    "@material-ui/core": "^3.5.1",
+    "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.1",
+    "@material-ui/lab": "^3.0.0-alpha.30",
```

Comment:
Where do you use the lab library? Please remove it if added by mistake.

Reply:
Thanks I removed. I added before when implementing toggle buttons, but it is unnecessary now.